File: .old-compat/src/main/java/androidx/preference/DialogPreference.java
package androidx.preference;

public abstract class DialogPreference extends Preference {
	public CharSequence title, message;
	
	public CharSequence getDialogTitle() {
		return title;
	}
	
	public void setDialogTitle(CharSequence dialogTitle) {
		this.title = dialogTitle;
	}
	
	public CharSequence getDialogMessage() {
		return message;
	}
	
	public void setDialogMessage(CharSequence dialogMessage) {
		this.message = dialogMessage;
	}
}

File: extension/loaders/android-compat/src/desktopMain/kotlin/androidx/preference/DialogPreference.kt
package androidx.preference

import android.content.Context
import com.mrboomdev.awery.core.utils.PlatformSdk

@PlatformSdk
abstract class DialogPreference(context: Context): Preference(context) {
	private var dialogTitle: CharSequence? = null
	private var dialogMessage: CharSequence? = null

	@PlatformSdk
	fun getDialogTitle(): CharSequence? {
		return dialogTitle
	}

	@PlatformSdk
	fun setDialogTitle(dialogTitle: CharSequence?) {
		this.dialogTitle = dialogTitle
	}

	@PlatformSdk
	fun getDialogMessage(): CharSequence? {
		return dialogMessage
	}

	@PlatformSdk
	fun setDialogMessage(dialogMessage: CharSequence?) {
		this.dialogMessage = dialogMessage
	}
}